addOns/scripts/src/main/javahelp/org/zaproxy/zap/extension/scripts/resources/help/contents/automation.html

```diff
@@ -43,6 +45,18 @@ <H2>Action: run</H2>
 	<li>target: mandatory, if type is 'targeted', the target URL to be invoked for 'targeted' script
 	</ul>
 
+	<H2>Action: loaddir</H2>
+
+	Loads all of the scripts in the subdirectories under the specified source path to ZAP. 
+	Scripts are enabled but not configured to be loaded when ZAP restarts.
+	<br><br>
```

Comment:
End paragraph?

Reply:
Updated, and the HTML I copied from above :)